app/assets/stylesheets/lifehack_style.scss

```diff
@@ -0,0 +1,118 @@
+.form_text {
+  float: left;
+  font-size: 15px;
+  font-weight: bold;
+  margin: 15px;
+}
+
+.lifehack-details {
+  max-width: 300rem;
+  padding: rem-calc(50);
+}
+
+  .panel {
+
+      h5 {
+
+        font-size: 30px;
+        text-align: center;
+      }
+      h2 {
+    border-style: double;
+    color: $primary-color;
+    font-size: 60px;
+    text-align: center;
+    }
+
+    table tr td {
+      background-color: $smoke;
+      padding: 13px 0.875rem 13px 2.875rem;
+      }
+
+    .creator-name {
+      padding: 0;
+      }
+
+    .creator-image {
+      padding: 5;
+      }
+
+    .lifehack-description {
+      font-size: 20px;
+    }
+
+    }
+
+  .description   {
+    p {
+      text-align: justify;
+      color: $primary-color;
+      font-size: 50px;
+      padding: 25px 15px 100px 20px;
+      width: 300px;
+    }
+
+    table tr th {
+      background-color: $smoke;
+      font-size: 18px;
+      margin-right: auto;
+      margin-left: auto;
+      padding: 3px 5.875rem 21px 4.875rem;
+      text-align: center;
+    }
+
+    table tr td {
+
+      background-color: $smoke;
+      font-size: 15px;
+      margin-right: auto;
```

Comment:
Properties should be ordered background-color, font-size, margin-left, margin-right, padding, text-align